flexmeasures/data/models/generic_assets.py

```diff
@@ -498,6 +498,89 @@ def chart(
 
         return chart_specs
 
+    def multiple_chart(
+        self,
+        chart_type: str = "chart_for_multiple_sensors",
+        event_starts_after: datetime | None = None,
+        event_ends_before: datetime | None = None,
+        beliefs_after: datetime | None = None,
+        beliefs_before: datetime | None = None,
+        source: DataSource
+        | list[DataSource]
+        | int
+        | list[int]
+        | str
+        | list[str]
+        | None = None,
+        include_data: bool = False,
+        dataset_name: str | None = None,
+        resolution: str | timedelta | None = None,
+        **kwargs,
+    ) -> list[dict]:
+        """Create a vega-lite chart showing sensor data.
```

Comment:
Several charts, one per entry in `sensors_to_show

Reply:
i changed it to  """Create multiple vega-lite charts, one chart per entry in `sensors_to_show`. ...